docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md

```diff
@@ -86,6 +86,10 @@ global:
         clientId: <Client ID from step 2>
         audience: <Audience from step 2>
         existingSecret: <Client secret from step 2>
+      webModeler:
+        clientId: <Client ID from step 2>
```

Comment:
Should there also be a `clientSercret` variable here?

Reply:
No, Web Modeler doesn't require a client secret.

Reply:
🤦🏻  of course